canonical-websites/cloud-init.io
# Update run script to version 0.3.2

## Done
Updated the run script to the latest version

## QA
- Pull branch
- Run `./run` and check it works
- Run `./run watch` and make a change and see it changes
- Run `./run clean` and see it cleans
